server/@types/ui/index.d.ts

```diff
@@ -0,0 +1,14 @@
+import { AccreditedProgramme } from '@accredited-programmes/models'
+
+export type SummaryListRow = {
+  key: {
+    text: string
+  }
+  value: {
+    text: string
+  }
+}
+
+export type ProgrammeListItem = Omit<AccreditedProgramme, 'programmePrerequisites'> & {
+  programmePrerequisites: Array<SummaryListRow>
```

Comment:
Sorry, I missed this on my original review. To avoid confusion in the nunjucks templates and to make this extra clear, could we call this key `programmePrerequisitesSummaryListRows` so it's clear it's some nunjucks-formatted objects?